src/d3d9/d3d9_constant_buffer.cpp

```diff
@@ -146,7 +146,7 @@ namespace dxvk {
     // this applies to is so large that it will not matter.
     if (pDevice->CanSWVP()
      && ShaderStage == DxsoProgramType::VertexShader
-     && BufferType == DxsoConstantBuffers::VSConstantBuffer)
+     && (BufferType == DxsoConstantBuffers::VSConstantBuffer || BufferType == DxsoConstantBuffers::VSVertexBlendData))
```

Comment:
Looking at FF code, looks like vertex blend data is aways a storage buffer regardless of SWVP state.

I'm honestly in favour of unconditionally setting these as `UNIFORM | STORAGE` since drivers don't give a single shit about buffer usage flags anyway.

Reply:
Done.